ord_schema/validations.py

```diff
@@ -136,7 +126,6 @@ def ensure_details_specified_if_type_custom(message):
     if message.type == message.CUSTOM and not message.details:
         warnings.warn(f'Custom type defined for {type(message)}, '
                       'but details field is empty', ValidationError)
-    return message
```

Comment:
I like the idea of separating validations and updates. Removing the `return message` from every validator is a nice bonus